src/main/resources/application.properties

```diff
@@ -12,4 +12,10 @@ logging.level.org.springframework.web=DEBUG
 logging.level.org.hibernate=ERROR
 logging.file=ocdu.log
 
-spring.thymeleaf.mode=LEGACYHTML5
\ No newline at end of file
+spring.thymeleaf.mode=LEGACYHTML5
+
+
+autonomous.cron.schedule=1 * * * * *
+autonomous.upload.source.directory=/home/jacob/tomcat/upload/source
```

Comment:
This path is specific to your setup - do you think we could replace it with some relative path that will work also on the server and our workstations?